File: Blackjack Project/Part 1A/name.cpp
#include<iostream>

using namespace std;

int main()
{
    int card;
    cin >> card;
    if(card == 1){
        cout << "Drew an Ace" << endl;
    } else if(card == 8){
        cout << "Drew an 8" << endl;
    } else if(card >= 2 && card <= 10){
        cout << "Drew a " << card << endl;
    } else if(card == 11){
        cout << "Drew a Jack" << endl;
    } else if(card == 12){
        cout << "Drew a Queen" << endl;
    } else if(card == 13){
        cout << "Drew a King" << endl;
    } else{
        cout << "BAD CARD" << endl;
    }

}

File: Blackjack Project/Part 1A/name.py
card = int(input())
if card == 1:
    print("Drew an Ace")
elif card == 8:
    print("Drew an 8")
elif card >= 2 and card <= 10:
    print("Drew a " + str(card))
elif card == 11:
    print("Drew a Jack")
elif card == 12:
    print("Drew a Queen")
elif card == 13:
    print("Drew a King")
else:
    print("BAD CARD")
